examples/tutorials/sst2_classification_non_distributed.py

```diff
@@ -0,0 +1,156 @@
+"""
+SST-2 Binary text classification with XLM-RoBERTa model
+=======================================================
+
+**Author**: `Parmeet Bhatia <parmeetbhatia@fb.com>`__
+
+"""
+
+######################################################################
+# Overview
+# --------
+#
+# This tutorial shows how to train a text classifier using pre-trained XLM-RoBERTa model.
+# TODO
+
+
+######################################################################
+# Common imports
+# --------------
+
+import torch
+import torch.nn as nn
+DEVICE = torch.DEVICE("cuda") if torch.cuda.is_available() else "cpu"
+
+
+#######################################################################
+# Data Transformation
+# -------------------
+# TODO
+
+import torchtext.transforms as T
+from torch.hub import load_state_dict_from_url
+
+PADDING_IDX = 1
```

Comment:
Since this file is now in the form more appropriate to tutorial, I think you can lowercase the most of variables.